dali/operators/expressions/arithmetic_meta.h

```diff
@@ -22,13 +22,16 @@
 #include <utility>
 
 #include "dali/core/cuda_utils.h"
+#include "dali/core/small_vector.h"
 #include "dali/core/static_switch.h"
 #include "dali/core/tensor_shape.h"
 #include "dali/pipeline/data/backend.h"
 #include "dali/pipeline/data/types.h"
 
 namespace dali {
 
+constexpr int kMaxArity = 2;
```

Comment:
Nitpick: it's only used once and not really in the context of limiting arity (the latter is explicitly checked to be 1 or 2, not <= something).

Reply:
Missed one spot. Now it's used in two SmallVectors and the next is coming in next PR.